source/overview.rst

```diff
@@ -93,6 +122,11 @@ In the pre-processing step, users need to create the WEC geometry, run a BEM cod
    * The ``*.stl`` files are used to visualize the WEC response in Simscape Explorer
    * They are also used for :ref:`nonlinear` forces if the option is enabled.
 
+.. note::
+    This bit seems a bit out of place because the stl file isn't actually used
+    for BEMIO, if my understanding is correct. Perhaps some recommendations
+    on making the CAD model (and it what software) would be more useful here.
+
 * **Compute Hydrodynamic Coefficients**: WEC-Sim requires frequency-domain hydrodynamic coefficients (e.g. added mass, radiation damping, and wave excitation). 
```

Comment:
This section has now been split into two steps: 1) pre-processing, and 2) generating the hydrodata file